docs/application/dotnet/guides/connectivity/webview.md

```diff
@@ -0,0 +1,166 @@
+
+<a name="initialize"></a>
+## Initializing and Finalizing WebView
+
+To Initialize and Finalize WebView, follow these steps:
+
+1. To use the `Tizen.WebView` namespace in your application, call the `Tizen.WebView.Chromium.Initialize` method before the start of the main loop, as shown in the following code snippet:
+
+    ```
+    Elementary.Initialize();
+    Elementary.ThemeOverlay();
+    Chromium.Initialize();
+    App app = new App();
+    app.Run(args);
+    ```
+
+2. To clean up the resources allocated by your application, call the `Tizen.WebView.Chromium.Shutdown` method after the end of the main loop, as shown in the following code snippet:
+
+    ```
+    App app = new App();
+    app.Run(args);
+    Chromium.Shutdown();
+    ```
+
+<a name="create"></a>
+## Creating WebView Object
+
+To create a WebView object, use the constructor of `Tizen.WebView.WebView` with `ElmSharp.Window`, as shown in the following code snippet:
+
+```
+var webView = new WebView(window)
+{
+    AlignmentX = -1,
+    AlignmentY = -1,
+    WeightX = 1,
+    WeightY = 1
+};
+webView.Show();
+```
+
+<a name="load"></a>
+## Loading Web Page
+
+To load a web page, use the `Tizen.WebView.WebView.LoadUrl` method with an appropriate URL, as shown in the following code snippet:
+
+```
+webView.LoadUrl("https://www.tizen.org/");
+```
+
+<a name='event'></a>
+## Handling Load Events
+
+The following table lists the loading events provided by the `Tizen.WebView.WebView` class:
+
+**Table: Loading Events**
+
+| Event            | Description                                                       | Argument                                 |
+|------------------|-------------------------------------------------------------------|------------------------------------------|
+| `LoadStarted`    | Event occurs when the application starts to load.                 | -                                        |
+| `LoadFinished`   | Event occurs when the application finishes loading.               | -                                        |
+| `LoadError`      | Event occurs when an error appears while loading the application. | Tizen.WebView.SmartCallbackLoadErrorArgs |
+
+- Add an event handler to handle each loading event, as shown in the following code snippet:
+
+    ```
+    webView.LoadStarted += (s, e) =>
+    {
+        /* Handle LoadStarted event here */
+    };
+
+    webView.LoadFinished += (s, e) =>
+    {
+        /* Handle LoadFinished event here */
+    };
+
+    webView.LoadError += (s, e) =>
+    {
+        /* Handle LoadError event here */
+
+        // error code
+        var code = e.Code;
+
+        // description
+        var desc = e.Description;
+
+        // URL
+        var url = e.Url;
+    };
+    ```
+
+<a name='cookie'></a>
+## Setting Cookie Options
+
+Almost all the web pages require cookies. To manage and set the cookies, use the [Tizen.WebView.CookieManager](https://samsung.github.io/TizenFX/latest/api/Tizen.WebView.CookieManager.html) class.
```

Comment:
I think adding a brief one sentence description of what cookie is would be good.

Reply:
I think we don't need to explain about the cookie here, because cookie is already well-known feature in web industry.
The reason why I've added this section is that the cookie is well-known and widely used feature, and many developer want to know how to manage the cookie for Tizen .Net application.

Reply:
@younghajung If you think we need not explain about cookies, that's fine. 
You can remove that statement too as we are removing the description.